Tests/UnitTests/Fatras/Digitization/UncorrelatedHitSmearerTests.cpp

```diff
@@ -69,8 +75,9 @@ struct InvalidSmearer {
   }
 };
 
+template <typename generator_t>
 struct Fixture {
-  RandomGenerator rng;
+  generator_t rng;
```

Comment:
why do we need to template the random engine?

Reply:
Ah this is because I use the Gaussian smearer for this test [](https://github.com/acts-project/acts/blob/main/Examples/Algorithms/Digitization/include/ActsExamples/Digitization/Smearers.hpp), which uses the `RandomEngine` and not the `RandomGenerator` that is the default in the `Fixture` struct. So I think it needs to change.
Maybe it is better to have as templated in order to use other smearers, too? If you agree with this approach

Reply:
What I might be missing is why we need a different RNG for the new test? Why does the `RandomGenerator` not work in this case?